platform/source/tools/clustercli/common.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015-2017 Applatix, Inc. All rights reserved.
+#
+
+import re
+
+from anytree import Node
+from .cli import AbstractPrompt
+
+
+class CommonPrompts(AbstractPrompt):
+
+    INSTALLCMD = "install"
+    UNINSTALLCMD = "uninstall"
+
+    def __init__(self):
+        self.aws_profiles = self._get_profiles()
+        self._profile_names = [x for x in self.aws_profiles]
+
+        self.root = Node("name",
+                    prompt=u'Enter a cluster name',
+                    validator=u'^[a-zA-Z0-9-_]{2,20}$',
```

Comment:
for name, first letter cannot be numbers and this regex does not take care of it I think

Reply:
Will fix